lib/driver.js

```diff
@@ -1,7 +1,7 @@
 import { BaseDriver, DeviceSettings, errors } from 'appium-base-driver';
 import { desiredCapConstraints } from './desired-caps';
 import logger from './logger';
-import commands from './commands/index';
+import commands from './commands';
```

Comment:
if we `export { commands };` in lib/commands/general, how come we can just import commands from ./commands`?

also what are 'extensions' doing? (From `lib/commands/execute` and `lib/commands/general`)

Reply:
I am following how other drivers are doing it.
There are a few files in commands folder that are exporting commands.
As for extensions, they are Webdriver commands specific to a platform.